docs/src/main/paradox/java/http/routing-dsl/directives/basic-directives/extractMatchedPath.md

```diff
@@ -0,0 +1,15 @@
+<a id="extractmatchedpath-java"></a>
+# extractMatchedPath
+
+## Description
+
+Extracts the request's matched path.
+
+This directive is used to acquire part of path matched during routing matching
+process. Similar but opposite to `RequestContext` unmatched path property.
```

Comment:
Wording could be slightly improved. Just steal + adapt from `extractUnmatchedPath`?